### DISCUSSION TITLE
Is there a way to reconfigure SDDM theme using Hyde?

### DISCUSSION THREAD

**Discussion by zenghost1** [prasanthrangan/hyprdots#1507] [Q&A]
I was wondering if it's possible to change the SDDM theme to the other of the two available during installation using Hyde scripts.

**Comment by kRHYME7** [comment]
` Hyde sddm theme ` 

Just follow the instructions if it gives error or don't recognize your Clone directory 


`Hyde-install --link --dir ~/HyDE ` ~HyDE is the clone dir

**Reply by fmolero69** [reply]
138 / 5.000
Coincidentally, I was going to ask the same question.
I have done what you indicated. but I don't see an option that allows me to change the SDDM login theme

**Reply by kRHYME7** [reply]
@fmolero69  
` Hyde sddm theme ` 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/7efedc0a-a9a5-4c50-af0f-03bf216a46cc)

**Reply by fmolero69** [reply]
Oh! Thank you

**Reply by kRHYME7** [reply]
` Hyde man ` to see available CLI options

**Comment by Crazygiscool** [comment]
The `Hyde sddm theme` no longer works on my installation, was it removed in recent versions? if so, what is the current way?
